babel.config.json

```diff
@@ -1,5 +1,14 @@
 {
-  "plugins": ["babel-plugin-styled-components"],
+  "plugins": [
+    "babel-plugin-styled-components",
+    [
+      "formatjs",
+      {
+        // keep in sync with lang/extract-messages.ts
```

Comment:
This is a comment and doesn't belong in a json file (same applies to yumlet where we have introduced it).
- [ ]  remove it or rather change the file to babel.config.js or rather  babel.config.ts (https://babeljs.io/blog/2023/02/20/7.21.0#experimental-support-for-cts-configuration-files) if that works without much trouble.